flexboxgrid-master/.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,17 @@
+### Briefly ( tweet size ) describe your issue and how you discovered it.
+
+...
+
+### Expected behavior
+
+...
+
+### Actual behavior
+
+...
+
+### Steps to reproduce behavior
+
+* OS:
+* Browser:
+* code/fiddle/pen url:
```

Comment:
there are a lot of these flexbox files that should not be installed directly into your root 

Instead, you can get flexbox in your project by adding this to `application.html.erb` 
`<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flexboxgrid/6.3.1/flexboxgrid.min.css" type="text/css" >`

HOWEVER, you shouldnt really even need flexbox, as its patterns are built into Foundation 6! my recommendation would be to remove it altogether